<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1498</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="articles">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>1221</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="topics_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>251</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_field">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Search for Papers</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="search_progress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="only_pdfs">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Only Direct PDFs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="download_all">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Download All</string>
   </property>
  </widget>
  <widget class="QLabel" name="empty_label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>280</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Topics List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Papers</string>
   </property>
  </widget>
  <zorder>articles</zorder>
  <zorder>topics_list</zorder>
  <zorder>search_field</zorder>
  <zorder>search_progress</zorder>
  <zorder>only_pdfs</zorder>
  <zorder>empty_label</zorder>
  <zorder>download_all</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
